rst_guide/advanced_syntax.rst

```diff
@@ -0,0 +1,1090 @@
+
+:glyphicon-console:`\ ` Code example
+''''''''''''''''''''''''''''''''''''
+.. rst-tabs::
+
+  .. tab-content:: tab1-dictionary
+    :title: input: RST
+
+    .. code-block:: rst
+
+      .. div:: row
+
+        .. div:: col-12 col-md-6
+
+          .. image:: /images/gallery/sample.jpeg
+            :alt: This is a sample image.
+            :width: 380px
+            :align: center
+            :class: img-responsive img-circle
+
+          This is not a caption, this is only a paragraph below the image.
+
+        .. div:: col-12 col-md-6
+
+          .. figure:: /images/gallery/sample.jpeg
+            :alt: This is a sample figure.
+            :scale: 10%
+            :align: center
+            :class: img-responsive img-thumbnail
+
+            **Image X.** This is the caption.
+
+  .. tab-content:: tab2-dictionary
+    :title: rendered: HTML
+
+    .. div:: html-box
+
+      :raw-html:`<div class="row">`
+      :raw-html:`<div class="col-12 col-md-6">`
+      :raw-html:`<a class="img-responsive img-circle reference internal image-reference" href="http://localhost:8080/static/default/_images/sample.jpeg"><img alt="This is a sample image." class="img-responsive img-circle align-center" src="http://localhost:8080/static/default/_images/sample.jpeg" style="width: 380px;"></a>`
+      :raw-html:`<p>This is not a caption, this is only a paragraph below the image</p>`
+      :raw-html:`</div>`
+      :raw-html:`<div class="col-12 col-md-6">`
+      :raw-html:`<div class="figure align-center" id="id8">`
+      :raw-html:`<a class="img-responsive img-thumbnail reference internal image-reference" href="http://localhost:8080/static/default/_images/sample.jpeg"><img alt="This is a sample figure." class="img-responsive img-thumbnail" src="http://localhost:8080/static/default/_images/sample.jpeg" style="width: 326.40000000000003px; height: 244.8px;"></a>`
+      :raw-html:`<p class="caption"><span class="caption-text"><strong>Image X.</strong> This is the caption.</span></p>`
+      :raw-html:`</div>`
+      :raw-html:`</div>`
+      :raw-html:`</div>`
+
+.. rst-class:: pull-right
+
+| :glyphicon-info-sign:`\ ` **Read more about**  `images <https://docutils.sourceforge.io/docs/ref/rst/directives.html#images>`_.
+
+|
+
+Dictionary can add a hover window |Sphinx| :term:`def term <Sphinx>`
+
+.. |Sphinx| replace:: :term:`def term <Sphinx>`
+
+.. glossary::
+
+  Sphinx
+    Sphinx is a tool that makes it easy to create intelligent and beautiful documentation. It was originally created for the Python documentation, and it has excellent facilities for the documentation of software projects in a range of languages.
+
+  Sublime Text
+    Sublime Text is a sophisticated text editor for code, markup and prose. You'll love the slick user interface, extraordinary features and amazing performance.
+
+Error management
+
+
+This is a standalone link https://docutils.sourceforge.io/docs/.
```

Comment:
Is this still part of the Dictionary section? Should there be a new heading here or what is this?